entity-framework/toc.yml

```diff
@@ -70,36 +70,10 @@
         href: core/get-started/wpf.md
       - name: Xamarin tutorial
         href: core/get-started/xamarin.md
-    - name: Fundamentals
-      items:
-      - name: Connection strings
-        href: core/miscellaneous/connection-strings.md
-      - name: Asynchronous programming
-        href: core/miscellaneous/async.md
-      - name: Logging
-        href: core/miscellaneous/logging.md
-      - name: Context pooling
-        href: core/miscellaneous/context-pooling.md
-      - name: Connection resiliency
-        href: core/miscellaneous/connection-resiliency.md
-      - name: Testing
-        items:
-        - name: Testing code that uses EF Core
-          href: core/miscellaneous/testing/index.md
-        - name: EF Core testing sample
-          href: core/miscellaneous/testing/testing-sample.md
-        - name: Sharing databases between tests
-          href: core/miscellaneous/testing/sharing-databases.md
-        - name: Test with SQLite
-          href: core/miscellaneous/testing/sqlite.md
-        - name: Test with InMemory
-          href: core/miscellaneous/testing/in-memory.md
       - name: Configure a DbContext
         href: core/miscellaneous/configuring-dbcontext.md
-      - name: Nullable reference types
-        href: core/miscellaneous/nullable-reference-types.md
-      - name: Collations and case sensitivity
-        href: core/miscellaneous/collations-and-case-sensitivity.md
+      - name: Connection strings
```

Comment:
This page might need some more work before it's front-and-center (#1730)

Reply:
Yeah, I agree.. But I think it's OK for it to be this way for a short time until we do #1730 (which I can also help with if needed). Just to not block.

BTW there's a chance a dedicated page on configuring connection strings isn't really needed, assuming our tutorials cover the different scenarios.